bindings/python/structured/grid.py

```diff
@@ -0,0 +1,321 @@
+#
```

Comment:
This seems to be an implementation in Python of the Grid, and next there's also index_space. Why do you need a python implementation instead of challenging to the C++ implementation?

Reply:
The Grid implementation here is completely superfluous and removed now. The Cartesian Sets are a neat way to work with domains and provide a nice interface to the user that goes beyond what the C++ implementation provides. To be fair they have far more features than strictly needed though. For example the IndexSpace is never used in the bindings themselves, but make the tests very short as a domain decomposition becomes just a single line. For now I would keep this and suggest we revisit this again when we have a better picture on how people use the bindings in their codes.